### PR TITLE
Added Ubuntu installation instructions

### PR DESCRIPTION
**Pull request by amura11**

### CODE REVIEW

File: content/en/docs/Installation/pre-built-binaries.md [hunk 1]
```diff
@@ -25,3 +25,119 @@ You can install Navidrome via the
...
+#### Create Configuration File
+
+In the working directory, `/var/lib/navidrome` create a new file named `navidrome.toml` with the following settings.
+
+```
+LogLevel = "INFO"
+BaseURL = ""
+ScanInterval = "10s"
+TranscodingCacheSize = "15MB"
+MusicFolder = "<library_path>"
```
Comment (deluan) [inline content/en/docs/Installation/pre-built-binaries.md:71]: I suggest keeping the config file as simple as possible, changing it to just:

```
MusicFolder = "<library_path>"
```

Comment (deluan) [reply content/en/docs/Installation/pre-built-binaries.md:71]: Based on your unit example, don't we need to set the DataFolder to `/var/lib/navidrome`?

Comment (deluan) [reply content/en/docs/Installation/pre-built-binaries.md:71]: NVM, just realized that you set the WorkingDirectory to it, should be fine, then 👍

Comment (amura11) [reply content/en/docs/Installation/pre-built-binaries.md:67]: The minimal config makes sense, I update that section and change some wording in a few places

**Comment by deluan** [comment]
Looks good, thanks you!

**Pull request merged**